pjsip-apps/src/pjsua/pjsua_app_legacy.c

```diff
@@ -318,7 +318,7 @@ static void vid_handle_menu(char *menuin)
         argv[argc] = strtok(NULL, " \t\r\n");
     }
 
-    if (argc == 1 || strcmp(argv[1], "help")==0) {
+    if (argc <= 1 || strcmp(argv[1], "help")==0) {
```

Comment:
since the command requires space, i.e. `vid `, argc can't be 0.
```
            if (menuin[1]=='i' && menuin[2]=='d' && menuin[3]==' ') {
                vid_handle_menu(menuin);
```